### ISSUE TITLE
Using six-group.com as source for automatic updates

### ISSUE THREAD

**Issue by courtney-miles** [alcohol/iso4217#30]
Would the package maintainers be interested in a PR that scripts the updating of the codes in this package?

The idea would be to use [SIX Financial Information](https://www.six-group.com/en/products-services/financial-information/data-standards.html) as the source.

> SIX Financial Information AG is the official Maintenance Agency of these currency codes under ISO 4217 and as such the only recognized, authoritative source on currency code designations.

They offer and XML copy of all the code current codes at https://www.six-group.com/dam/download/financial-information/data-center/iso-currrency/lists/list_one.xml

In regards to #4, they also offer a list of historic denominations at https://www.six-group.com/dam/download/financial-information/data-center/iso-currrency/lists/list_three.xml

*(I have taken an interest because this package lists ZWL as deprecated when it's otherwise still in the official list, including on Wikipedia -- so it seems less ambiguity is needed)*

**Comment by alcohol** [comment]
Yes, though I can't quite say what I would like this to look like. But I am definitely open to scripted updating. If you can start with a PR with some very basic things, I can perhaps provide early feedback.

**Comment by alcohol** [comment]
I also am still not sure if I want to make this library more in line with how I setup https://github.com/thephpleague/iso3166 or not (I sometimes prefer the more "simple" approach I still have here).

**Comment by alcohol** [comment]
It would probably be best if the update script could dump the parsed xml contents into a generated file with php code (e.g. `<?php return [...];`) to allow for opcode caching.
